File: Linguagem C#/Repetitiva/DentroFora.cs
using System;

namespace DentroFora {
    class Program {
        static void Main(string[] args) {

            int n, x,contDentro,contFora;

            Console.Write("Quantos numeros voce vai digitar? ");
            n = int.Parse(Console.ReadLine());

            contDentro = 0;
            contFora = 0;
            for (int i= 1;i<=n;i++) {
                Console.Write("Digite um numero: ");
                x = int.Parse(Console.ReadLine());
                
                if (x >= 10 && x <= 20) {
                    contDentro++;
                }else {
                    contFora++;
                };
            };

            Console.WriteLine(contDentro+" DENTRO");
            Console.WriteLine(contFora+" FORA");
        }
    }
}

File: Linguagem Python/Repetitiva/DentroFora.py
n = int(input("Quantos numeros voce vai digitar? "))
n = int(input("Quantos numeros voce vai digitar? "))

for i in range(1,n+1):
    x = int(input("Digite um numero: "))
    if x % 2 == 0 and x > 0:
        print("PAR POSITIVO")
    elif x % 2 == 0 and x < 0:
        print("PAR NEGATIVO")
    elif x % 2 != 0 and x > 0:
        print("IMPAR POSITiVO")
    elif x % 2 != 0 and x < 0:
        print("IMPAR NEGATIVO")
    else:
        print("NULO")